build.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/sh
-# Script to build ndt-server with the correct `go get` flags.
+# Script to build ndt-server with the correct `go get` flags.  This script
+# should be run inside a container.
```

Comment:
Why should?

Reply:
Hm. How about: "This script is designed and tested to run as part of the container build process"?

It's not forbidden to just run it, but I don't want to make any promises either.